PluralKit.Core/Database/Migrations/14.sql

```diff
@@ -0,0 +1,15 @@
+﻿-- SCHEMA VERSION 14: 2020-04-01 --
+-- added reminders table
+
+CREATE TABLE IF NOT EXISTS reminders
+(
+    mid bigint NOT NULL PRIMARY KEY,
+    channel bigint NOT NULL,
+    guild bigint,
+    member integer REFERENCES members (id) ON DELETE CASCADE,
+    system integer NOT NULL,
+    seen boolean NOT NULL,
```

Comment:
Not super important, but it'd probably be more correct to default this to `false` so you don't have to provide it in AddReminder.